packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -550,87 +602,101 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   const variables: CSSOutput['variables'] = [];
 
   // Quasis are the string pieces of the template literal - the parts around the interpolations
-  const literalResult = node.quasis.reduce<string>((acc, quasi, index): string => {
-    const nodeExpression = node.expressions[index] as t.Expression | undefined;
-
-    if (
-      !nodeExpression ||
-      (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
-    ) {
-      const suffix = meta.context === 'keyframes' ? '' : ';';
-      return acc + quasi.value.raw + suffix;
-    }
+  const literalResult = node.quasis.reduce<string>(
+    (acc: string, quasi: t.TemplateLitteral, index: number): string => {
```

Comment:
Git Actions is just saying you have a typo `t.TemplateLitteral`

Reply:
Turns out `TemplateLitteral` was the wrong type 😖  updated to `Templateelement`